tasks/broad/GermlineVariantDiscovery.wdl

```diff
@@ -130,7 +140,7 @@ task HaplotypeCaller_GATK4_VCF {
   runtime {
     docker: gatk_docker
     preemptible: preemptible_tries
-    memory: "6.5 GiB"
+    memory: "~{memory_size} GiB"
```

Comment:
You are absolutely right that changing the memory in the task inputs will break call caching for the task. We generally don't worry about this issue since you would rarely manually change the memory on a task that was passing (and therefore could benefit from call caching). It is important to note though!

As far as a variable affecting Cromwell's memory retry feature, I don't know the answer to this one. I don't see any indication in the documentation that this would be a problem, but I have reached out to the batch team to confirm. I will update here when I have an answer

Reply:
Batch has confirmed that this will **not** affect the memory retry feature in cromwell

Reply:
So this doesn't need to be changed and I can resolve this comment, correct?

Reply:
correct!